nano/node/active_transactions.cpp

```diff
@@ -49,7 +51,7 @@ void nano::active_transactions::confirm_frontiers (nano::transaction const & tra
 	// To minimise dropping real-time transactions, set the maximum number of elections
 	// for cementing frontiers to half the total active election maximum.
 	const auto max_active = node.config.active_elections_size / 2;
-	if (roots_size <= max_active && (check_time_exceeded || (!is_test_network && low_active_elections)))
+	if (roots_size <= max_active && (check_time_exceeded || (!is_test_network && low_active_elections && !lazy_mode)))
 	{
 		// When the number of active elections is low increase max number of elections for setting confirmation height.
 		if (max_active > roots_size + max_elections)
```

Comment:
So as I understand it:
- Lazy mode, every 3 minutes confirm up to half of max active elections (25k) 
- Auto (not enough weight) - Don't confirm 
- Auto (with enough weight) - Confirm every 15 minutes (or if there is a very low number of active elections) up to half max active elections.
- Always - Same as above.
Is this correct?